CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+- **BREAKING** Add support for additional chip select pins for `lpspi4`.
```

Comment:
I've squashed, signed, and pushed. The revision history looks clean and consistent, the two operations now neatly separated.

I wasn't quite sure about the CHANGELOG, and I probably should have asked about it. I wasn't sure that the changes wouldn't conflict with existing code that works around this issue, and it does change the interface slightly. In retrospect, I might have gone with a "minor" change. Let me know if you change your mind on this and would rather have it listed as minor; t's easy to fix.

Otherwise, merge at will.